app/quest-boost/[boostId]/page.tsx

```diff
@@ -0,0 +1,195 @@
+"use client";
+
+import React, { useContext, useEffect, useState } from "react";
+import styles from "../../../styles/questboost.module.css";
+import { Call, CallDetails } from "starknet";
+import {
+  getBoostById,
+  getQuestBoostClaimParams,
+  getQuestParticipants,
+  getQuestsInBoost,
+  updateQuestBoostClaimStatus,
+} from "../../../services/apiService";
+import Quest from "../../../components/quests/quest";
+import { useRouter } from "next/navigation";
+import { QuestDocument } from "../../../types/backTypes";
+import Timer from "../../../components/quests/timer";
+import { useAccount, useContractWrite } from "@starknet-react/core";
+import Button from "../../../components/UI/button";
+import boostContractCalls from "../../../utils/calldata/boost_contract_claim";
+import { StarknetIdJsContext } from "../../../context/StarknetIdJsProvider";
+import { useNotificationManager } from "../../../hooks/useNotificationManager";
+import {
+  NotificationType,
+  TransactionType,
+} from "../../../constants/notifications";
+import { hexToDecimal } from "../../../utils/feltService";
+import { CDNImage } from "../../../components/cdn/image";
+
+type BoostQuestPageProps = {
+  params: {
```

Comment:
Wy do you need an object inside the object and add boostId right away ?

Reply:
Since migrating to next 14
we pass page props inside a `params` key.
something like this https://github.com/starknet-id/starknet.quest/blob/171e9f91928e7628d10cc52b260f7330f2f68d72/app/categories/%5Bcategory%5D/page.tsx#L12